airflow/api_fastapi/core_api/routes/ui/dashboard.py

```diff
@@ -44,17 +44,18 @@
     responses=create_openapi_http_exception_doc([status.HTTP_400_BAD_REQUEST]),
 )
 def historical_metrics(
-    start_date: DateTimeQuery,
-    end_date: DateTimeQuery,
     session: Annotated[Session, Depends(get_session)],
+    start_date: DateTimeQuery,
+    end_date: OptionalDateTimeQuery | None = None,
```

Comment:
I think `OptionalDateTimeQuery` already holds that information.

```suggestion
    end_date: OptionalDateTimeQuery = None,
```